apis/cache/v1alpha1/cachebackend_types.go

```diff
@@ -38,11 +41,20 @@ type CacheBackendSpec struct {
 
 // CacheBackendStatus defines the observed state of CacheBackend
 type CacheBackendStatus struct {
-	// JobName indicates the name of the job that consumes the cache
-	JobName string `json:"jobName,omitempty"`
+	// CacheEngine is the current cache engine used by CacheBackend
+	CacheEngine string `json:"cacheEngine,omitempty"`
 
 	// CacheStatus is used to help understand the status of a caching process
 	CacheStatus CacheStatus `json:"cacheStatus,omitempty"`
+
+	// UsedBy array contains the jobs currently using this cacheBackends
+	UsedBy []string `json:"usedBy,omitempty"`
+
+	// UsedNum equals to the size of UsedBy array
+	UsedNum int `json:"usedNum,omitempty"`
```

Comment:
since UsedNum equals to the size of UsedBy array, why we need maintain another filed?